packages/amplify_analytics_plugin_interface/pubspec.yaml

```diff
@@ -9,8 +9,9 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  plugin_platform_interface: ^1.0.1
   meta: ^1.1.8
+  amplify_core:
+    path: ../amplify_core
```

Comment:
cc: @haverchuck. I'm confused about when to use local pathing vs. not. If you use a version number, does melos publish to and read from some local development store, using those numbers? How to I publish to that development store, to prevent the need for the paths?

Reply:
When using `melos` we don't need local paths as running `melos bootstrap` in the root directory will setup proper  local symlinks for the dependencies and won't look for pub.get for them.

However the gotcha is that you won't be able to run `flutter pub get` in the individual packages since that will override the melos symlinks and will start using the `pub.get` versions (what you observed @jamesonwilliams  that local changes were not being picked up)

Reply:
@Amplifiyer I guess I can see the utility of being able to run pub get on individual packages during development, but why commit this file with a local path?

Reply:
unfortunately this is required to run iOS unit tests in the pipeline until that package is available from pub.get (or even after that) since `flutter pub get` is necessary to be run before running iOS tests and running that basically overrides everything melos did and so iOS tests then run on the published versions and not the local dependencies.

This is the same issue we ran into amplify-js where we resolved it by using local publishing via verdaccio to use local package dependencies. We may need to find a solution for iOS tests here to possibly run without requiring `flutter pub get` (it's currently required to create/update flutter's Podfile)

Reply:
Ah, I see.  So this is a temporary pipeline workaround for a new package?